Objects/bytearrayobject.c

```diff
@@ -170,9 +185,10 @@ PyByteArray_AsString(PyObject *self)
     return PyByteArray_AS_STRING(self);
 }
 
-int
-PyByteArray_Resize(PyObject *self, Py_ssize_t requested_size)
+static int
+_PyByteArray_Resize_lock_held(PyObject *self, Py_ssize_t requested_size)
```

Comment:
```suggestion
bytearray_resize_lock_held(PyObject *self, Py_ssize_t requested_size)
```
Avoid Py prefix if it is static function.

Reply:
Sorry, missed that in the merge of recent main.